src/main/java/com/Studentlifr/GratitudeBoard/services/MessageService.java

```diff
@@ -0,0 +1,15 @@
+package com.Studentlifr.GratitudeBoard.services;
+
+import com.Studentlifr.GratitudeBoard.dao.MessageDAO;
+import com.Studentlifr.GratitudeBoard.dto.requests.MessageReqDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.GenericResDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.PublicMessagesResDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.ResponseDTO;
+
+import java.util.List;
+
+public interface MessageService {
+    public ResponseDTO addMessage(MessageReqDTO messageDTO, String userId);
```

Comment:
remove public here as any method declared here is public by default

Reply:
Removing Public from there, restricting me to access the message service package in controller, (Outside the Package).